src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-expressroute-circuit/main.combined.bicep

```diff
@@ -0,0 +1,18 @@
+resource expressRouteCircuit 'Microsoft.Network/expressRouteCircuits@2020-11-01' = {
+  name: 'expressRouteCircuit'


```

Comment:
@emilguden - can we get these carriage returns cleaned up?

Reply:
Fixed it! It was generated from when I ran SetBaseline.ps1 and forgot to remove all carriage returns..

Reply:
Fixed it!